cis_v150/section_4.sp

```diff
@@ -0,0 +1,472 @@
+  description   = "Enable Vulnerability Assessment (VA) setting 'Also send email notifications to admins and subscription owners'."
+  sql           = query.sql_server_va_setting_reports_notify_admins.sql
+  documentation = file("./cis_v150/docs/cis_v150_4_2_5.md")
+
+  tags = merge(local.cis_v150_4_2_common_tags, {
+    cis_item_id = "4.2.5"
+    cis_level   = "1"
+    cis_type    = "automated"
+    service     = "Azure/SQL"
+  })
+}
+
+benchmark "cis_v150_4_3" {
+  title         = "4.3 PostgreSQL Database Server"
+  documentation = file("./cis_v150/docs/cis_v150_4_3.md")
+  children = [
+    control.cis_v150_4_3_1,
+    control.cis_v150_4_3_2,
+    control.cis_v150_4_3_3,
+    control.cis_v150_4_3_4,
+    control.cis_v150_4_3_5,
+    control.cis_v150_4_3_6,
+    control.cis_v150_4_3_7,
+    control.cis_v150_4_3_8
+  ]
+
+  tags = merge(local.cis_v150_4_3_common_tags, {
+    type = "Benchmark"
+  })
+}
+
+control "cis_v150_4_3_1" {
+  title         = "4.3.1 Ensure 'Enforce SSL connection' is set to 'ENABLED' for PostgreSQL Database Server"
+  description   = "Enable SSL connection on PostgreSQL Servers."
+  sql           = query.postgres_sql_ssl_enabled.sql
+  documentation = file("./cis_v150/docs/cis_v150_4_3_1.md")
+
+  tags = merge(local.cis_v150_4_3_common_tags, {
+    cis_item_id = "4.3.1"
+    cis_level   = "1"
+    cis_type    = "automated"
+    service     = "Azure/PostgreSQL"
+  })
+}
+
+control "cis_v150_4_3_2" {
+  title         = "4.3.2 Ensure Server Parameter 'log_checkpoints' is set to 'ON' for PostgreSQL Database Server"
+  description   = "Enable log_checkpoints on PostgreSQL Servers."
+  sql           = query.postgres_db_server_log_checkpoints_on.sql
+  documentation = file("./cis_v150/docs/cis_v150_4_3_2.md")
+
+  tags = merge(local.cis_v150_4_3_common_tags, {
+    cis_item_id = "4.3.2"
+    cis_level   = "1"
+    cis_type    = "automated"
+    service     = "Azure/PostgreSQL"
+  })
+}
+
+control "cis_v150_4_3_3" {
+  title         = "4.3.3 Ensure server parameter 'log_connections' is set to 'ON' for PostgreSQL Database Server"
+  description   = "Enable log_connections on PostgreSQL Servers."
+  sql           = query.postgres_db_server_log_connections_on.sql
+  documentation = file("./cis_v150/docs/cis_v150_4_3_3.md")
+
+  tags = merge(local.cis_v150_4_3_common_tags, {
+    cis_item_id = "4.3.3"
+    cis_level   = "1"
+    cis_type    = "automated"
+    service     = "Azure/PostgreSQL"
+  })
+}
+
+control "cis_v150_4_3_4" {
+  title         = "4.3.4 Ensure server parameter 'log_disconnections' is set to 'ON' for PostgreSQL Database Server "
+  description   = "Enable log_disconnections on PostgreSQL Servers."
+  sql           = query.postgres_db_server_log_disconnections_on.sql
+  documentation = file("./cis_v150/docs/cis_v150_4_3_4.md")
+
+  tags = merge(local.cis_v150_4_3_common_tags, {
+    cis_item_id = "4.3.4"
+    cis_level   = "1"
+    cis_type    = "automated"
+    service     = "Azure/PostgreSQL"
+  })
+}
```

Comment:
The title and the description of both the controls look similar, please take a look.